src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -340,37 +334,16 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_send(MPIDI_OFI_SENDPARAMS)
     int mpi_errno;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_MPI_SEND);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_SEND);
-    mpi_errno = MPIDI_OFI_send(MPIDI_OFI_SENDARGS, 1, 0ULL);
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_NM_MPI_SEND);
-    return mpi_errno;
-}
 
-#undef FUNCNAME
-#define FUNCNAME MPIDI_NM_rsend
-#undef FCNAME
-#define FCNAME MPL_QUOTE(FUNCNAME)
-MPL_STATIC_INLINE_PREFIX int MPIDI_NM_rsend(MPIDI_OFI_SENDPARAMS)
-{
-    int mpi_errno;
-    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_RSEND);
-    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_RSEND);
-    mpi_errno = MPIDI_OFI_send(MPIDI_OFI_SENDARGS, 1, 0ULL);
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_NM_RSEND);
-    return mpi_errno;
-}
+    if (MPIDI_OFI_ENABLE_TAGGED) {
```

Comment:
@wesbland It looks like here should be unversed logic because [MPIDIG_mpi_send func](https://github.com/pmodels/mpich/blob/master/src/mpid/ch4/generic/mpidig_send.h#L135) calls am send but MPIDI_OFI_send works with tagged API. I'm not 100% sure but looks like the same for Recv path